.github/workflows/buf-delete.yaml

```diff
@@ -0,0 +1,16 @@
+name: Buf delete
```

Comment:
I assume this is deleting a label (would indicate in the name)

Reply:
The `buf-pr` is for a pull request event, so this `buf-delete` is for a delete event but the action is to archive the label.

Reply:
yeah makes sense, I figured that out and thought I deleted this coment